### PR TITLE
Archaeology-Junimo Fixes

### PR DESCRIPTION
**Pull request by Witchybun**
- Reduce initial values for bartering.
  - Junimos go from 100% -> 50% to 65% -> 15%.
  - Bear go from 100% -> 20% to 85% -> 15%.
- Fix Corn/Sunflower/Wheat warning
- Fix Archaeology names.
  - Reintroduction of a translation dictionary (sigh).
  - New method which replaces incorrect item names.

### CODE REVIEW

File: StardewArchipelago/Stardew/StardewItemManager.cs [hunk 1]
```diff
@@ -519,9 +519,9 @@ private void AddObjectByColor(ObjectData objectData, StardewObject stardewObject
                 return; //Its an object with no tags attached, so wouldn't have a color.
             }
             var firstColor = objectData.ContextTags.FirstOrDefault(x => x.Contains("color_"));
-            if (firstColor is null)
+            if (firstColor is null || Constants.Modded.IgnoredModdedStrings.JojaRouteSVE.Contains(stardewObject.Name))
```
Comment (agilbert1412) [inline StardewArchipelago/Stardew/StardewItemManager.cs:522]: I think the items should make it to the list, and get filtered upon usage instead.
StardewItemManager should, as much as possible, hold all the data available, not a filtered version of it

Comment (agilbert1412) [reply StardewArchipelago/Stardew/StardewItemManager.cs:522]: Plus, I'm not a fan of introducing mod-related code in that file

File: StardewArchipelago/Constants/Modded/IgnoredModdedStrings.cs [hunk 2]
```diff
@@ -33,5 +33,10 @@ public class IgnoredModdedStrings
             "Restore Prismatic Shard", "Restore Golden Mask", "Restore Ancient Sword", "Restore Ancient Doll", "Restore Chipped Amphora",
         };
 
+        public static readonly List<string> JojaRouteSVE = new()
+        {
+            "Razor Trout", "Joja Berry", "Joja Berry Starter", "Joja Veggie", "Joja Veggie Seeds"
```
Comment (agilbert1412) [inline StardewArchipelago/Constants/Modded/IgnoredModdedStrings.cs:38]: Trailing comma
image

File: StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs [hunk 3]
```diff
@@ -94,6 +94,7 @@ private void CheckAllShipsanityLocations(List<Item> allShippedItems)
                 {
                     continue;
                 }
+                name = FixArchaeologyLocationInconsistentNaming(name);
```
Comment (agilbert1412) [inline StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs:97]: This should happen in the Archaeology section of NameSimplifier, not in Shipsanity code

File: StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs [hunk 4]
```diff
@@ -105,5 +106,17 @@ private void CheckAllShipsanityLocations(List<Item> allShippedItems)
                 }
             }
         }
+
+        private string FixArchaeologyLocationInconsistentNaming(string itemName)
```
Comment (agilbert1412) [inline StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs:110]: And the method should move there too

File: StardewArchipelago/GameModifications/Modded/JunimoShopStockModifier.cs [hunk 5]
```diff
@@ -20,7 +20,10 @@ public class JunimoShopStockModifier : BarterShopStockModifier
         private static readonly string[] spring = new string[]{"spring"};
         private static readonly string[] summer = new string[]{"summer"};
         private static readonly string[] fall = new string[]{"fall"};
+        private static readonly string[] summer_fall = new string[]{"summer", "fall"};
         private static readonly string[] winter = new string[]{"winter"};
+        private static float INITIAL_DISCOUNT = 0.65f;
```
Comment (agilbert1412) [inline StardewArchipelago/GameModifications/Modded/JunimoShopStockModifier.cs:25]: These two can be consts

**Review by agilbert1412 — CHANGES_REQUESTED** [review]

File: StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs (hunk 3 above)
Comment (Witchybun) [reply StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs]: Done.  In fact, the old code stopped working as the internal names changed to be more normal.  They just aren't the same as the english display name.

File: StardewArchipelago/Stardew/StardewItemManager.cs [hunk 6]
```diff
@@ -519,7 +519,7 @@ private void AddObjectByColor(ObjectData objectData, StardewObject stardewObject
                 return; //Its an object with no tags attached, so wouldn't have a color.
             }
             var firstColor = objectData.ContextTags.FirstOrDefault(x => x.Contains("color_"));
-            if (firstColor is null || Constants.Modded.IgnoredModdedStrings.JojaRouteSVE.Contains(stardewObject.Name))
+            if (firstColor is null)
             {
                 return; // There was no color tag found or its from the Joja route; throw it out.
```
Comment (agilbert1412) [inline StardewArchipelago/Stardew/StardewItemManager.cs:524]: Update the comment too plz

**Review by agilbert1412 — APPROVED** [review]

**Review by agilbert1412 — APPROVED** [review]

**Pull request merged**